docs/reference/content/driver-async/index.md

```diff
@@ -10,16 +9,20 @@ type = "index"
 
 ## MongoDB Async Java Driver Documentation
 
-The following guide provides information on using the
-MongoDB Async Java Driver 3.4.
+The following guide provides information on using the callback based MongoDB Async Java Driver 3.4.
```

Comment:
It should be, but if it's not consistent with the rest of the docs then leave it alone.